### ISSUE TITLE
[Bug] WCAG (Web Content Accessibility Guildlines) 2.1 Keyboard Accessibility

### ISSUE THREAD

**Issue by kevingostomski** [kevingostomski/better-nice-select#6]
Currently the plugin supports all possible navigation and selecting options for a complete keyboard accessible plugin and even more features via keyboard like the special optgroup label selector but there is missing one user control functionality:

- logic for keypress `Esc key` is missing
> No matter which element in the popup has focus, the popup should be closed when the key `Esc` gets pressed to fully support the keyboard accessibility feature to be WCAG compatible
